src/Stratis.FederatedPeg.Features.FederationGateway/SourceChain/RestSenderBase.cs

```diff
@@ -42,6 +49,7 @@ protected async Task<HttpResponseMessage> SendAsync<T>(T model, string route)
                 }
                 catch (Exception ex)
                 {
+                    this.backOffUntil = DateTime.Now.AddSeconds(10);
```

Comment:
probably an overkill for now, but something to keep in mind if the retry policy gets a bit more complex:
https://alastaircrabtree.com/implementing-the-retry-pattern-using-polly/

Reply:
Thanks. Will keep that in mind.